features/support/driver.js

```diff
@@ -166,9 +166,15 @@ class Driver {
     return browser.quit()
   }
 
+  async getCurrentUrlName () {
+    const browser = this.browser
+    const url1 = await browser.getCurrentUrl()
```

Comment:
Instead of creating the constant url1, you could just have the following:
**return browser.getCurrentUrl()** 

Please note that you don't need await when you are returning a promise.

Reply:
created async await function